cinema/serializers.py

```diff
@@ -0,0 +1,26 @@
+from rest_framework import serializers
+
+from cinema.models import Movie
+
+
+class MovieSerializer(serializers.Serializer):
+    id = serializers.IntegerField(read_only=True)
+    title = serializers.CharField(max_length=55)

```

Comment:
why do you have different max_length values in the model and serializer?

Reply:
yes, my mistake